src/mango_httpd.erl

```diff
@@ -20,6 +20,7 @@
 
 -include_lib("couch/include/couch_db.hrl").
 -include("mango.hrl").
+-include("mango_idx.hrl").
```

Comment:
Don't do this. Use the getter methods in mango_idx.erl to access the fields. Letting the record definition leak out makes it a lot harder to do upgrades later on.